test/functional/wallet_basic.py

```diff
@@ -421,7 +421,7 @@ def run_test(self):
             assert prebalance > 2
             address = self.nodes[1].getnewaddress()
             # Throw if no conf_target provided
-            assert_raises_rpc_error(-8, "Selected estimate_mode requires a fee rate",
+            assert_raises_rpc_error(-8, "Selected estimate_mode BTc/Kb requires a fee rate to be specified in conf_target",
```

Comment:
I suppose I'll make them all proper-cased in wallet_basic, and lowercased in fundrawtransaction, so they are less jarring while still testing the case insensitivity.

Reply:
Oh wait, in wallet_basic.py I'd rather not touch the units in the original tests by @kallewoof. They were set this way on purpose. See closed PR #20041 that wanted to change them.

Reply:
I'll add "case insensitive" to the logging so it's clear to future readers of the test code.

Reply:
Cool, thanks for digging up the context. I still think it would be better to test case insensitivity of an argument in one dedicated test and then make all the other tests use the canonical writing, but it's definitely out of scope here.

Reply:
Agree, and in a way that's what we've done: The added tests use canonical units and the original tests in wallet_basic.py do case insensitive unit testing and are now described as such.